src/bcf/record.rs

```diff
@@ -191,6 +256,12 @@ impl Record {
         self.push_format(tag, data, htslib::BCF_HT_REAL)
     }
 
+    /// Update genotypes for all samples, corresponds to `bcf_update_genotypes` in HTSlib.
+    pub fn push_format_genotypes(&mut self, gts: &[i32]) -> Result<(), TagWriteError> {
```

Comment:
Maybe this should rather be `set_genotypes(&mut self, gts: &[Genotypes])` and take a [Genotypes](https://docs.rs/rust-htslib/0.17.0/rust_htslib/bcf/record/struct.Genotypes.html) object? I know, this requires some additional implementation work.

Reply:
I'm removing this for now, adding as Issue in #67.